src/main/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherTaxServiceImpl.java

```diff
@@ -80,4 +86,43 @@ protected boolean validateNRATaxInformation(DisbursementVoucherDocument document
         }
         return true;
     }
+    
+    @Override
+	public boolean isForeignVendorAndTaxReviewRequired(String payeeTypeCode,
+			String paymentReasonCode, Integer vendorHeaderId) {
+
+		return isForeignVendor(payeeTypeCode, vendorHeaderId) && !paymentReasonDoesNotRequireTaxReviewForForeignVendor(paymentReasonCode);
+	}
+	
+    @Override
+	public boolean isForeignVendorAndTaxReviewNotRequired(String payeeTypeCode,
+			String paymentReasonCode, Integer vendorHeaderId) {
+
+		return isForeignVendor(payeeTypeCode, vendorHeaderId) && paymentReasonDoesNotRequireTaxReviewForForeignVendor(paymentReasonCode);
+	}
+	
+	protected boolean isForeignVendor(String payeeTypeCode, Integer vendorHeaderId){
+		return KFSConstants.PaymentPayeeTypes.VENDOR.equalsIgnoreCase(payeeTypeCode) && vendorService.isVendorForeign(vendorHeaderId);
```

Comment:
Nancy found a bug in this code. I changed it from payeeTypeCode.equals(KFSConstants.PaymentPayeeTypes.VENDOR) to KFSConstants.PaymentPayeeTypes.VENDOR.equalsIgnoreCase(payeeTypeCode) so that there is no longer a null pointer exception when an empty DV doc is submitted.